proposals/kroxylicious-operator-api-alternative.md

```diff
@@ -223,35 +232,19 @@ spec:
 
   # Points to the cluster being proxied.  Can either be Strimzi Kafka resource or endpoint details.
   targetCluster:
-    # one of: resourceRef or bootstrapping
-    resourceRef:
-      kind: Kafka  # must be Kafka
-      group: strimzi.io # must be strimzi.io
+    clusterRef:
+      group: strimzi.io|kroxylicious.io # must be strimzi.io or kroxylicious.io
+      kind: Kafka|KafkaClusterRef  # must be Kafka (belonging to group strimzi.io) or TargetClusterRef (belonging to group kroxylicious.io)
       name: my-cluster
-      listenerName: listener # name of strimzi listener 
-    bootstrapping:
-       bootstrap: bootstrap:9092
-       protocol: TCP|TLS
-       nodeIdRanges:
-       - name: mybrokers
-         range:
-          startInclusive: 0
-          endExclusive: 3
+      listenerName: listener # name of strimzi listener, permitted only for Kafka (group strimzi.io)
```

Comment:
Should be marked as optional right? Is it required for Strimzi references or can we default it?

Reply:
Yeah, it is optional.

Reply:
It might also be optional if the Kafka object has only one listener, or perhaps, if there is only one on-cluster listener.